recipes/configs/quant_generate.yaml

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
I dont think we need a separate config here. We should make it clear in our documentation and tutorial on how to load a quantized model. Its just a checkpointer change. So Id remove this